hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RocksDBConfiguration.java

```diff
@@ -48,17 +48,17 @@ public class RocksDBConfiguration {
 
   @Config(key = "rocksdb.max.log.file.size",
       type = ConfigType.SIZE,
-      defaultValue = "0MB",
+      defaultValue = "100MB",
       tags = {OM, SCM, DATANODE},
       description = "Maximum size of RocksDB application log file.")
-  private long rocksdbMaxLogFileSize = 0;
+  private long rocksdbMaxLogFileSize = 100;
```

Comment:
@Gargi-jais11 , here LONG "100" doesn't equal to size of "100MB".  

Suggest slow down,  carefully check the context, clearly understand the impact of the change, before submitting the patch.